src/fdc3/dotnet/DesktopAgent/tests/DesktopAgent.Tests/TestUtils/MockAppDirectory.cs

```diff
@@ -0,0 +1,263 @@
+﻿/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+using MorganStanley.Fdc3.AppDirectory;
+
+namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Tests.TestUtils;
+
+public class MockAppDirectory : IAppDirectory
```

Comment:
Don't mock types that are not owned by the current project (in this case, the solution). IAppDirectory already has an implementation that is tested and can load itself from JSON.